code/datums/mutations/olfaction.dm

```diff
@@ -40,6 +40,10 @@
 		to_chat(owner, span_warning("You have no nose!"))
 		return FALSE
 
+	if(HAS_TRAIT(living_cast_on, TRAIT_ANOSMIA)) //Anosmia quirk holder cannot fell any smell
+		to_chat(owner, span_warning("You can't feel any smells!"))
+		return FALSE
```

Comment:
I won't copy paste it across these but I think `You can't smell!` is maybe a better feedback message than `You can't feel any smells!`